### PR TITLE
Test cleanup

### LINKED ISSUE #684: Diganoze why a couple of tests are failing only for Ubuntu-devel combo
```r
== Failed tests ================================================================
-- Error ('test-MCMCglmm.R:20'): (code run outside of `test_that()`) -----------
Error in `terms.formula(formula, data = data)`: invalid term in model formula
Backtrace:
    x
 1. \-MCMCglmm::MCMCglmm(...) at test-MCMCglmm.R:20:2
 2.   \-stats::get_all_vars(...) at MCMCglmm/R/MCMCglmm.R:70:0
 3.     +-stats::terms(formula, data = data)
 4.     \-stats::terms.formula(formula, data = data)
-- Error ('test-survreg.R:8'): (code run outside of `test_that()`) -------------
Error in `survreg(Surv(time, status) ~ ph.ecog + age + strata(sex), data = lung)`: data contains an infinite predictor
Backtrace:
    x
 1. \-survival::survreg(...) at test-survreg.R:8:2

```

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
- https://github.com/easystats/easystats/issues/350
- https://github.com/easystats/easystats/issues/343

### CODE REVIEW

**Comment by IndrajeetPatil** [comment]
Probably you already know, but you can minimize the changes to review by hiding whitespace changes.

Screenshot 2023-03-18 at 16 19 42

**Review by strengejacke — APPROVED** [review]

**Comment by strengejacke** [comment]
> Probably you already know, but you can minimize the changes to review by hiding whitespace changes.
> 
> Screenshot 2023-03-18 at 16 19 42

No, didn't know that. Nice feature!

File: tests/testthat/test-get_modelmatrix.R [hunk 1]
```diff
@@ -59,7 +57,11 @@ test_that("get_modelmatrix - iv_robust", {
 # ====================================================================
 
 test_that("get_modelmatrix - ivreg", {
+  skip_if(getRversion() < "4.0.0")
+  skip_if_not_or_load_if_installed("ivreg")
   data(Kmenta, package = "ivreg")
+
+  set.seed(15)
```
Comment (strengejacke) [inline tests/testthat/test-get_modelmatrix.R:64]: Does this seed lead to the failing test on old-rel?

**Review by strengejacke — CHANGES_REQUESTED** [review]

File: tests/testthat/test-get_modelmatrix.R (hunk 1 above)
Comment (IndrajeetPatil) [reply tests/testthat/test-get_modelmatrix.R:64]: No, I added it to fix the failing test, but it didn't do anything. No clue why unloading AER namespace leads these tests to fail.

Comment (strengejacke) [reply tests/testthat/test-get_modelmatrix.R:64]: AER includes the "predecessor" of `ivreg`, maybe `AER::ivreg()` and `ivreg::ivreg()` give slightly different results on R 4.0? Maybe try `ivreg::ivreg()`, and see if this works (or fails only on one R version, and if so, skip all until oldrel (and older)?)

Comment (IndrajeetPatil) [reply tests/testthat/test-get_modelmatrix.R:64]: I just skipped the test in question on older R version.

**Review by strengejacke — APPROVED** [review]

**Pull request merged**